### DISCUSSION TITLE
v2.10.1

### DISCUSSION THREAD

**Discussion by mienaiyami** [mienaiyami/yomikiru#100] [General]
## Downloads

[![Download setup](https://img.shields.io/badge/Windows%20Setup%20(exe)-Yomikiru--2.10.1--Setup.exe-brightgreen?logo=windows&logoColor=blue)](https://github.com/mienaiyami/yomikiru/releases/download/v2.10.1/Yomikiru-2.10.1-Setup.exe)

[![Download Portable](https://img.shields.io/badge/Windows%20Portable%20(zip)-Yomikiru--win32--2.10.1--Portable.zip-brightgreen?logo=windows&logoColor=blue)](https://github.com/mienaiyami/yomikiru/releases/download/v2.10.1/Yomikiru-win32-2.10.1-Portable.zip)

---
## Note

- **Epub reader available (experimental)**

Linux distribution currently not available as direct download but can be build easily by following this [Build for linux(debian) guide](https://github.com/mienaiyami/yomikiru/blob/master/docs/build_for_linux.md).

---

## Added

- EPUB: Auto scroll to old position after setting changes.
- Option to open epub from file explorer.

## Fixed

- #98.

---

#### Todo

- Improve Epub reader.
- Write tests.
- Add ability to change image quality in canvas mode for less RAM usage and performance.

---

**Full Changelog**: https://github.com/mienaiyami/yomikiru/compare/v2.10.0...v2.10.1

<hr /><em>This discussion was created from the release <a href='https://github.com/mienaiyami/yomikiru/releases/tag/v2.10.1'>v2.10.1</a>.</em>

**Comment by omodia** [comment]
- I found this alignment issue of top buttons while opening settings, cycling through themes, clicking on home button.
- I did not open an issue immediately, because I don't know if it is a bug or normal behaviour.
- Is it a bug?

**Reply by mienaiyami** [reply]
System buttons are not part of app and i can only give it height and background color data, there might be some glitch as i dont know of anything that will cause this.
Try pressing ctrl + 0 then test or try restarting. Make new issue if it persists.

**Reply by omodia** [reply]
It does persist after trying what you mentioned. But will it affect any RAM usage?

In v 2.10.0, because of the side list bug, the RAM usage was 20 - 30 MB more than v2.10.1 for the same chapter.

**Reply by mienaiyami** [reply]
> But will it affect any RAM usage?

No, it has been in app for few months now
